deploy/rhsm-eph-clowdenv.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: v1
```

Comment:
Would you mind adding something like this that kind of documents why this file exists
```suggestion
# Derived from https://github.com/RedHatInsights/bonfire/blob/master/bonfire/resources/ephemeral-cluster-clowdenvironment.yaml
# but with the token-refresher sidecar enabled
apiVersion: v1
```

Reply:
:+1: